### ISSUE TITLE
Who is going to build network of receivers?

### ISSUE THREAD

**Issue by darekzbik** [UAVTracking/UAVTrackingProtocol#23]
Are there any plans for building network of receivers? Who will build install and operate these receivers? Are you going to use OGN? Is OGN sufficient if you would like to sue some reckless UAV operator?

**Comment by pjalocha** [comment]
I presume, to start with, the OGN network would be engaged.
But there are questions here: would the OGN network send drones positions onto OGN APRS or would it only demodulate/error-correct packets and forward them raw to the server, where the authentication would be verified ?
I understand an OGN reciever is not able to verify the packet signature, as it does not know the secret key of any drone.

**Comment by snip** [comment]
I think the signature can be sent os APRS field on OGN network

**Comment by pjalocha** [comment]
If the signature is 32-bit per position this is no problem, but to verify the signature you need as well the original packet, thus the OGN receiver would need to send the whole packet in its original binary form.

**Comment by darekzbik** [comment]
I'm happy to hear comments from OGN network guys. My question was not technical and not related to technical details like packet size. It is more related to legal, financial stuff and also in some way to coverage. 
As government would like to force UAV to send some signal probably they have idea who will receive and and how it will be used. I love OGN network but it is run by volunteers and stations can be turn off at any time. Also packets can be filtered when station operator decided to do so.

@yohanboniface can you comment what is your (official) point of view.

**Comment by darekzbik** [comment]
@pjalocha In case of asymmetric signatures OGN stations and OGN tracers will be able to verify signatures.  MACs versus asymmetric key signatures is also discussed in other issues.

**Comment by yohanboniface** [comment]
> @yohanboniface can you comment what is your (official) point of view.

Sadly it's a bit more complex than my point of view being the official point of view, but I can share what is the vision of the DINSIC (the State IT department, to make it simple, for which I work).
Our idea is that the network should behave more or less like Internet does: there is a bunch of open protocols that establishes a common language, and then there are servers, local network and such that are ran by either business companies, administrations, private entities… Some are open source based, some are proprietary, but when they use the network, they use that common language.
We see the "UAVs/gliders/small aircrafts" tracking network in the same way: we try to push an open source free to use common set of protocols (that would include this very one), that then may be used either by business companies or administration or geeks or…
Now some receivers can use the collected data only in private, others can share it again (like OGN would); some transmitters may only send the minimal info without any additional service, while others may build more service on top of it (like telemetry for RC models, collision avoidance, find a stolen/lost UAV…).
Here OGN could do a lot of things: being the common open infrastructure, being the wiki with open info on how to build a transmitter/receiver, etc.
In our vision, the French administration would deploy receivers that forward data to OGN, and to the French national servers.
But I'm not the one that takes the final decision on this. We (the DINSIC) are here just trying to make that step open and collaborative, because we believe it's the only way to converge toward a common protocol.

**Comment by ClLTL** [comment]
It may even be more complex... We know that drones will have to detect and avoid other aircraft, including other drones. So, it would be useful to consider both the Air to ground use of the protocol (being tracked by a ground network) and the air to air use of it (seeing other aircraft to keep will clear of them). May be drones will have to avoid all other aircraft in the future regulation as they are too small to be visually acquired by manned aircraft's pilots. It would then be a pity to have drones able to transmit their position to the ground but not allowed to fly because they are not able to detect other aircraft / to receive their cooperative messages...

**Comment by darekzbik** [comment]
On ISM band drones are able to 'see' other aircrafts. Unfortunately it is
rather impossible to force all planes to transmit possision in the same
way. Gliders are probably the easiest as lot of them has FLARM.

For sure there will be two difficult groups of airspace users. The first
one consists of small planes, ultralight and paragliders. The second one
will be airlines.

Pilots if a small planes can be convinced as taking small radio device will
clear drones from their way. For bigger planes it could be more difficult
and probably it will require global law regulation.
I don't know what paragliders will do. Is there any paraglider pilot here
who can comment?

**Comment by pjalocha** [comment]
A small ADS-B receiver is not that difficult to build (in contrast to the ADS-B transmitter), thus if airliners would carry ADS-B (already most of them do), the gliders, drones, etc. would easily see them.

**Comment by darekzbik** [comment]
As you know ADS-B not always send your position. And receiving it requires
changes in radio it is no longer ISM band.

**Comment by pjalocha** [comment]
Yes, I mean a totally independent ADS-B receiver, for short range it is really not a difficult thing, as the signals are real strong ! For general safety and avoidance of airlines by drones, gliders and all other "small" air traffic, it would be really nice if such a receiver in a small and light form would be available. I think it is worth to launch an R&D in this area.

**Comment by pjalocha** [comment]
Possibly there is already such a thing: https://www.unmannedtechshop.co.uk/ping-rx-ads-b-receiver-for-uavs/ - the text there suggests the autopilot would indeed avoid received aircrafts.
